src/components/Notifications/NotificationTypes/Transfer/GiftNotification.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import { i18n } from "./GiftNotification.i18n"
+import { GiftIcon } from "../../../Icon"
+import { NotificationItem } from "../../NotificationItem"
+import {
+  NotificationItemDescription,
+  NotificationItemTitle,
+} from "../../NotificationItem.styled"
+import { CommonNotificationProps, GiftNotificationProps } from "../../types"
+import { replaceWithValues } from "../../utils"
+
+const GiftNotification = React.memo(
+  (props: CommonNotificationProps<GiftNotificationProps>) => {
+    const { notification, locale, renderProfile } = props
+    const sender =
+      renderProfile?.(notification.metadata.senderAddress) ??
+      notification.metadata.senderAddress
+    return (
+      <NotificationItem
+        image={<GiftIcon width={48} height={48} />}
+        timestamp={notification.timestamp}
+        isNew={!notification.read}
+        locale={locale}
+      >
+        <NotificationItemTitle>
+          {replaceWithValues(i18n[locale].title, { sender })}
```

Comment:
I'm not sure how it's done in other notifications, but if we've got the profile component, we might want to use it to render the name and the user's avatar, WDYT?